src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -799,4 +799,22 @@ Reason: {2}</value>
   <data name="SuppressDiagnosticWithCommentNonNestedCodeActionTitle" xml:space="preserve">
     <value>Suppress the {0} diagnostic with Acuminator in a comment</value>
   </data>
-</root>
+  <data name="PX1078MessageFormat_TypesOfDacFieldAndReferencedFieldMismatch" xml:space="preserve">
+    <value>DAC field "{0}" and referenced field "{1}" have different types. Expected type for the DAC field is {2}.</value>
```

Comment:
@EAndrosova please review the messages for new diagnostic PX1078.
I believe, current messages should look like this:

- Different CLR types: 
```
DAC field "PaymentTermsID" and referenced field "SubstitutionID" have different types. Expected type for the DAC field is String.
```
- Different lengths: 
```
DAC field "PaymentTermsID" and referenced field "SubstitutionID" have different length specified in data type attribute. Expected length for DAC field is 25.
```
Incompatible type annotation: 
```
DAC field "PaymentTermsID" have data type attribute(s) incompatible with type "String" of referenced field "SubstitutionID"
```

Reply:
@EAndrosova please check

Reply:
Remark from @EAndrosova:
```suggestion
    <value>The "{0}" DAC field and the "{1}" referenced field have different types. The expected type for the DAC field is {2}.</value>
```